lib/thread.h

```diff
@@ -240,6 +240,12 @@ extern void thread_call (struct thread *);
 extern unsigned long thread_timer_remain_second (struct thread *);
 extern struct timeval thread_timer_remain(struct thread*);
 extern int thread_should_yield (struct thread *);
+
+extern unsigned long timeval_elapsed (struct timeval a, struct timeval b);
```

Comment:
@sli35 We've moved the time functions to monotime.h as part of a generic cleanup effort that is going on.  @qlyoung is correct these functions belong there.